services/movie.py

```diff
@@ -28,13 +34,21 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
```

Comment:
Consider using type hints for the `genres_ids` and `actors_ids` parameters, similar to how it's done in the `get_movies` function. This will improve code readability and consistency.